src/Common/src/Interop/Windows/Crypt32/Interop.CertCloseStore.cs

```diff
@@ -9,7 +9,7 @@ internal static partial class Interop
 {
     internal static partial class Crypt32
     {
-        [DllImport(Interop.Libraries.Crypt32, SetLastError = true)]
+        [DllImport(Interop.Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
This charset was specified in X509 version and most other Interop imports in common, I think it might have been left out

Reply:
Since there are no strings in the signature it doesn't really matter either way.

Reply:
Cool, guess I'll add them just for consistency's sake